be2-scala/src/main/scala/ch/epfl/pop/model/objects/Channel.scala

```diff
@@ -51,6 +51,7 @@ object Channel {
   final val rootChannel: Channel = Channel(s"${SEPARATOR}root")
   final val rootChannelPrefix: String = s"${SEPARATOR}root${SEPARATOR}"
   private final def channelRegex: String = "^/root(/[^/]+)*$"
+  final val LAODATALOCATION: String = Channel.SEPARATOR + "data"
```

Comment:
Any particular reason why this path is here, it's kind of odd to (to me at least) to see this in a Channel class.

Reply:
Yes, we discussed this with @RaulinN in my other PR, the Channel path values should be in channel instead of being defined in "random" places in the code, it's better to group everything.